integrationsupport/setup.go

```diff
@@ -25,20 +25,27 @@ func SetupTest(t *testing.T, testContext mmrtesting.TestContext, testGenerator T
 	tenantID := mmrtesting.DefaultGeneratorTenantIdentity
 
 	// Generate an initial batch of events. These are the last known backed-up events by the user.
-	GenerateTenantLog(&testContext, testGenerator, 7, tenantID, true, TestMassifHeight)
+	eventsToAppend := 7
```

Comment:
These changes are because the later versions of golangci lint for go 1.23 get annoyed at magic numbers outside of test code -.-